recipes-support/sqlite/sqlite3_%.bbappend

```diff
@@ -0,0 +1 @@
+CFLAGS:append = " -DSQLITE_ENABLE_DBSTAT_VTAB"
```

Comment:
What is this for?  Maybe a small comment would help here.

Reply:
I've added a comment explaining that this is needed for the auth cache in libocpp for OCPP 2.0.1